lib/absinthe/federation/schema/entities_field.ex

```diff
@@ -171,6 +176,23 @@ defmodule Absinthe.Federation.Schema.EntitiesField do
     end
   end
 
+  defp convert_key_to_atom(map) when is_map(map) do
```

Comment:
plural makes a bit more sense to me since it is recursive
```suggestion
  defp convert_keys_to_atom(map) when is_map(map) do
```

Reply:
it makes sense